docker/nuxeo-customer-project-sample-docker/README.md

```diff
@@ -0,0 +1,18 @@
+nuxeo/nuxeo-explorer Docker Image
+===================================
+
+This module is responsible to build the Explorer Docker image.
```

Comment:
```suggestion
This module is responsible to build the customer project's Docker image.
```

Reply:
@ataillefer @akervern actually it's "responsible *for* building"